cmd/prober/write.go

```diff
@@ -151,24 +168,23 @@ func rekorWriteEndpoint(ctx context.Context, cert *x509.Certificate, priv *ecdsa
 		verificationCounter.With(prometheus.Labels{verifiedLabel: verified}).Inc()
 	}()
 
-	body, err := rekorEntryRequest(cert, priv)
-	if err != nil {
-		return fmt.Errorf("rekor entry: %w", err)
-	}
-	req, err := retryablehttp.NewRequest(http.MethodPost, hostPath, bytes.NewBuffer(body))
-	if err != nil {
-		return fmt.Errorf("new request: %w", err)
-	}
-
-	setHeaders(req, "")
-
-	t := time.Now()
-	resp, err := retryableClient.Do(req)
-	latency := time.Since(t).Milliseconds()
+	resp, t, err := makeRekorRequest(cert, priv, hostPath)
 	if err != nil {
 		return fmt.Errorf("error adding entry: %w", err)
 	}
 	defer resp.Body.Close()
+	// A new body should be created when it is conflicted
+	for {
+		if resp.StatusCode != http.StatusConflict {
+			break
+		}
+		resp, t, err = makeRekorRequest(cert, priv, hostPath)
+		if err != nil {
+			return fmt.Errorf("error adding entry: %w", err)
+		}
+		defer resp.Body.Close()
+	}
+	latency := time.Since(t).Milliseconds()
```

Comment:
Can we record latency and response data for each request to Rekor? This would measure the total time for all requests.